setp.sh

```diff
@@ -0,0 +1,4 @@
+# This file will not be tracked by git as it's declared in .gitignore
```

Comment:
This is obviously not true.

Reply:
First, I use GH web editor. Second, there's `git add --force` that force stages a file even if it's in .gitignore.